extensions/interactions/TextInput/directives/text-input-rules.service.ts

```diff
@@ -29,51 +29,55 @@ import { TextInputRuleInputs } from 'interactions/rule-input-defs';
 export class TextInputRulesService {
   constructor(private nws: NormalizeWhitespacePipe) {}
   Equals(answer: TextInputAnswer, inputs: TextInputRuleInputs): boolean {
-    var normalizedAnswer = this.nws.transform(answer);
-    var normalizedInput = this.nws.transform(inputs.x);
-    return normalizedAnswer.toLowerCase() === normalizedInput.toLowerCase();
+    const normalizedAnswer = this.nws.transform(answer).toLowerCase();
+    const normalizedInput = inputs.x.map(
+      input => this.nws.transform(input).toLowerCase());
+    return normalizedInput.includes(normalizedAnswer);
   }
   FuzzyEquals(answer: TextInputAnswer, inputs: TextInputRuleInputs): boolean {
-    var normalizedAnswer = this.nws.transform(answer);
-    var answerString = normalizedAnswer.toLowerCase();
+    const normalizedAnswer = this.nws.transform(answer).toLowerCase();
+    const normalizedInput = inputs.x.map(
+      input => this.nws.transform(input).toLowerCase());
 
-    var normalizedInput = this.nws.transform(inputs.x);
-    var inputString = normalizedInput.toLowerCase();
-
-    if (inputString === answerString) {
-      return true;
-    }
-    var editDistance = [];
-    for (var i = 0; i <= inputString.length; i++) {
-      editDistance.push([i]);
-    }
-    for (var j = 1; j <= answerString.length; j++) {
-      editDistance[0].push(j);
-    }
-    for (var i = 1; i <= inputString.length; i++) {
-      for (var j = 1; j <= answerString.length; j++) {
-        if (inputString.charAt(i - 1) === answerString.charAt(j - 1)) {
-          editDistance[i][j] = editDistance[i - 1][j - 1];
-        } else {
-          editDistance[i][j] = Math.min(
-            editDistance[i - 1][j - 1], editDistance[i][j - 1],
-            editDistance[i - 1][j]) + 1;
+    const levenshtein = (inputString, matchString) => {
```

Comment:
Start with a verb, and also be clearer that this is returning a boolean rather than the distance: hasEditDistanceEqualToOne or isDifferentByOneCharacter or hasExactlyOneTypo or similar.